src/slider/internal.tsx

```diff
@@ -0,0 +1,188 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState, useRef } from 'react';
+import clsx from 'clsx';
+
+import styles from './styles.css.js';
+import { getBaseProps } from '../internal/base-component/index.js';
+import { SliderProps } from './interfaces.js';
+import useBaseComponent from '../internal/hooks/use-base-component/index.js';
+import { fireNonCancelableEvent } from '../internal/events/index.js';
+import { useFormFieldContext } from '../internal/context/form-field-context.js';
+import { useUniqueId } from '../internal/hooks/use-unique-id/index.js';
+import Tooltip from '../internal/components/tooltip/index.js';
+import SliderLabels from './slider-labels.js';
+import { getPercent, getStepArray, findLowerAndHigherValues, valuesAreValid, THUMB_SIZE } from './utils.js';
+import customCssProps from '../internal/generated/custom-css-properties/index.js';
+import { warnOnce } from '@cloudscape-design/component-toolkit/internal';
```

Comment:
nit: move this up to the other external dependencies. (I wish we had an eslint plugin for import order...)